osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -795,43 +841,19 @@ namespace renderkit {
         }
         checkForGLError("RenderManagerOpenGL::RenderDisplayInitialize end");
 
-		// Store the frame buffer that was active before we started rendering,
-		// so we can put it back when we finalize.
-		GLint fb;
-		glGetIntegerv(GL_FRAMEBUFFER_BINDING, &fb);
-		m_initialFrameBuffer = static_cast<GLuint>(fb);
-
 		return true;
     }
 
     bool RenderManagerOpenGL::RenderEyeInitialize(size_t eye) {
         checkForGLError("RenderManagerOpenGL::RenderEyeInitialize starting");
 
         // Render to our framebuffer
-        glBindFramebuffer(GL_FRAMEBUFFER, m_frameBuffers.at(GetDisplayUsedByEye(eye)));
+        glBindFramebuffer(GL_FRAMEBUFFER, m_frameBuffers.at(eye));
```

Comment:
This was in here to make possible the case where the client wants to render both eyes into the same frame buffer.  Is this being handled differently now, or does this change need to be reverted?

Reply:
Looks like the framebuffer is being bound below based on whether the eyes are the same.  Do we still need this binding here?

Reply:
So, I'm not sure exactly what is going on here because `m_frameBuffers` is created in `constructRenderBuffers` for the `Render` path and it always creates one framebuffer per eye, which seems to indicate that `m_frameBuffers.at(eye)` is the correct way to get the framebuffer. But, the comment in the header for `m_frameBuffers` seems to indicate it is "one per display", but that's not what the code is doing when it creates the frameBuffers. This code here (prior to the change) and the code in the OpenGL/D3D11 interop backend bind to the framebuffers in `m_frameBuffers` uses DISPLAY indices not eye indices, so in the end I'm not sure it's even working?

The code below that binds the frame buffer is binding to the framebuffer for a given display (always one per display) using the toolkit override if available, or else defaults to 0. This is probably technically wrong for multiple displays but usually ends up working because we usually have multiple contexts when using multiple displays and therefor the display framebuffer is typically 0 for a given display's context. We should probably have a more robust solution but that's maybe beyond the scope of this PR.

Reply:
I'm going to revert this change as it seems to not be related to the optimization work and we can revisit the Render path another time.

Reply:
Good; the GetDisplayUsedByEye() was in there exactly to handle the different cases of one or two eyes per display.  It was tested and worked in all cases when it was put in there.